src/dmd/backend/cgobj.d

```diff
@@ -1568,39 +1590,40 @@ private void objheader(char *csegname)
   }
   objrecord(COMENT,comment.ptr,comment.sizeof);
 
-    // Send out comment indicating we're using extensions to .OBJ format
```

Comment:
why change 4 spaces to two? Ditto throughout.

Reply:
Hum right, I didn't realize most of the function was misindented.

Reply:
Fixed